2.4/run-mongod.sh

```diff
@@ -6,6 +6,8 @@ source /var/lib/mongodb/.bashrc
 
 set -eu
 
+export HOME=/var/lib/mongodb
```

Comment:
I made this change because I'd made a similar change to the mysql image. However, after trying it out without the change to the script above, everything worked ok. So I've removed it. Updated the commit.